Lib/collections/__init__.py

```diff
@@ -1038,9 +1038,9 @@ def __repr__(self):
         return f'{self.__class__.__name__}({", ".join(map(repr, self.maps))})'
 
     @classmethod
-    def fromkeys(cls, iterable, *args):
-        'Create a ChainMap with a single dict created from the iterable.'
-        return cls(dict.fromkeys(iterable, *args))
+    def fromkeys(cls, iterable, value=None, /):
```

Comment:
Technically, it is a breaking change. `iterable` could be passed as a keyword argument: `ChainMap.fromkeys(iterable=['a', 'b', 'c'])`. Changing it to positional-only requires a deprecation period, and using more complex code meanwhile.

I think that's too much, this change shouldn't break much user code. But you should ask other core developers if they don't mind making an exception for this case. Some core developers have more hard position.

Reply:
This is pretty minor and ChainMap.fromkeys() is an almost never used backwater. I'm not worried about this.